packages/app/app/components/FavoriteTracksView/index.tsx

```diff
@@ -34,6 +36,13 @@ const FavoriteTracksView: React.FC<FavoriteTracksViewProps> = ({
   removeFavoriteTrack
 }) => {
   const { t } = useTranslation('favorites');
+  const dispatch = useDispatch();
+
+  const addToQueue = useCallback(( tracks: Track[]) => {
```

Comment:
I think this should instead add all tracks to the queue, and enable shuffle, instead of shuffling the order when adding

Reply:
wow that was a fast response haha. oh I didn't know you can enable shuffle mode while in your queue, will look into this. 

In terms of adding the button, are there any guidelines to the UI?

Reply:
Not really, you can take a look at other existing buttons. And please remember to create tests.